### ISSUE TITLE
The "spouse" link on some people goes back to the same person

### ISSUE THREAD

**Issue by trentlarson** [DecentralizedGenealogy/webclient#2]
On some of the individuals, the "spouse" link just goes back to the same person and not the spouse.  (Using Firefox.)

For example:
https://decentralizedgenealogy.github.io/webclient/www/person.html?id=https://raw.githubusercontent.com/misbach/familytree/master/people/KWCJ-RNC/KWCJ-RNC.json
https://decentralizedgenealogy.github.io/webclient/www/person.html?id=https://raw.githubusercontent.com/misbach/familytree/master/people/KWCG-VG1/KWCG-VG1.json

**Comment by misbach** [comment]
yeah, thanks for the bug report. There are many areas that need work. I'd
love to get some help if you are interested?

Matt <https://TreeSeek.com/>
